shared/src/main/scala/pl/metastack/metarx/Dep.scala

```diff
@@ -0,0 +1,19 @@
+package pl.metastack.metarx
+
+/** @see [[Sub.dep()]] */
+private[metarx] class Dep[T, U](sub: Sub[T],
+                                fwd: ReadChannel[T] => ReadChannel[U],
+                                bwd: ReadChannel[U] => ReadChannel[T]) {
+  private var value: U = _
+
+  private val internal = Sub[U](fwd(sub))
+  internal.attach(u => value = u)
+
+  sub := sub.get
```

Comment:
Is there a better way to accomplish set update of `value` without having to do this?